internal/sdkprovider/service/kafka/kafka_connector_test.go

```diff
@@ -265,7 +265,7 @@ data "aiven_project" "foo" {
 resource "aiven_kafka" "bar" {
   project                 = data.aiven_project.foo.project
   cloud_name              = "google-europe-west1"
-  plan                    = "startup-2"
+  plan                    = "business-4"
```

Comment:
This used to work on a cheaper plan. Or does it fail with the new checks?

Reply:
Hmm right, I'm not sure how this worked before, as the resource specified `kafka_connect = true` which is not supported in startup-2. Even the old comment says: "nosemgrep: kafka connectors need kafka with business plans"